csi/moac/pool.ts

```diff
@@ -236,7 +244,7 @@ class Pool {
         `Failed to list new replica "${uuid}" on pool "${this}": ${err}`
       );
     }
-    var replicaInfo = resp.replicas.filter((r) => r.uuid === uuid)[0];
+    var replicaInfo = resp.replicas.filter((r: any) => r.uuid === uuid)[0];
```

Comment:
r: any should be r: Replica?

Reply:
it's slightly confusing but replica here is object with replica properties returned by grpc method - not Replica object that represents replica internally in moac. So we must use any until we take a look at how to represent grpc replies by TS types.